### ISSUE TITLE
Web component: Masthead - Skip to main content link is missing from the page.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#3897]
## Detailed description

Issue: On Keyboard tab/shift tab navigation, Skip to main content link is missing from the page.

Expected: Skip to main content link should be available

## Additional information

Actual 
![Skip2MainCon.gif](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/5c98e70b-0611-43cf-8307-8d546463e023)

Expected:
![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/9ef1410e-bb4e-4e32-8ac7-154bc8c76abb)
